chunks.go

```diff
@@ -170,7 +170,13 @@ func (w *chunkWriter) finalizeTail() error {
 	if err := tf.Truncate(off); err != nil {
 		return err
 	}
-	return tf.Close()
+
+	if err := tf.Close(); err != nil {
+		return err
+	}
+
+	// close dir file (if not windows platform will fail on rename)
+	return w.dirFile.Close()
```

Comment:
I don't believe this is correct. `finalizeTail` is called whenever we move on to the next chunk file. Thus after the first chunk file is completed we'd end up with a closed file handle for `dirFile`.

Which rename does this attempt to fix?

Reply:
oh！
How about  move it too chunks.Close?

Reply:
Yes, that should be good.